src/compute/src/lib.rs

```diff
@@ -70,6 +70,13 @@ pub struct ComputeNodeOpts {
     #[clap(long, default_value = "")]
     pub config_path: String,
 
+    /// URI for tiered cache.
+    /// "none://" and "file://<path>" are supported.
+    /// Note: Currently, "file://<path>" is only supported on linux.
+    /// Setting "file://<path>" on other targets will be ignored.
+    #[clap(long, default_value = "none://")]
+    pub tiered_cache_uri: String,
```

Comment:
I think the `tiered_cache_uri` is more like a detailed config option for file cache and does not have to be specified in the launch command. IMO a more elegant config can be the following for storage config unless we are going to support more cache tiers at a lower level than file cache.
```
enable_file_cache: true,
file_cache_dir: /.../...,
```